<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoAudioPreferences</class>
 <widget class="QDialog" name="VideoAudioPreferences">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="resolution_label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Resolution: </string>
   </property>
  </widget>
  <widget class="QComboBox" name="video_resolution">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>50</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>N/A</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>144p</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>240p</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>360p</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>480p</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>720p</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1080p</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1440p</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1800p</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2160p</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="frame_rate_label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Frame Rate: </string>
   </property>
  </widget>
  <widget class="QComboBox" name="frame_rate">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>90</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>N/A</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>24 fps</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>30 fps</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>60 fps</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="video_bitrate_label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>120</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Video Bitrate: </string>
   </property>
  </widget>
  <widget class="QLabel" name="video_codec_label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>160</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Video Codec: </string>
   </property>
  </widget>
  <widget class="QSlider" name="video_br_slider">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>120</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maximum">
    <number>4</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TickPosition::TicksAbove</enum>
   </property>
   <property name="tickInterval">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="video_bitrate_level">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>140</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::LeftToRight</enum>
   </property>
   <property name="text">
    <string>N/A</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="audio_bitrate_label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>280</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Audio Bitrate:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="audio_bitrate">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>280</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>N/A</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>64 kbps</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>128 kbps</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>192 kbps</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>256 kbps</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>320 kbps</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="audio_codec_label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>320</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Audio Codec: </string>
   </property>
  </widget>
  <widget class="QLabel" name="sample_rate_label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>200</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Sample Rate: </string>
   </property>
  </widget>
  <widget class="QLabel" name="audio_channels_label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>240</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Channels: </string>
   </property>
  </widget>
  <widget class="QComboBox" name="video_codec">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>160</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>N/A</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>H.264</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>H.265</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>VP9</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>AV1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>FFV1</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="audio_codec">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>320</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>N/A</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>AAC</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>MP3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>OPUS</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>FLAC</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>PCM</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="sample_rate">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>200</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>N/A</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8000 Hz</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>22050 Hz</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>44100 Hz</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>48000 Hz</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>96000 Hz</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>192000 Hz</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="channels">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>240</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>N/A</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Mono</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Stereo</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5.1 Surround</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7.1 Surround</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="save_preferences">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>420</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Preferences</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_preferences">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>420</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel Preferences</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
